test/TransFNeg.ll

```diff
@@ -0,0 +1,41 @@
+; ModuleID = 'neg.bc'
+source_filename = "neg.cl"
+target datalayout = "e-i64:64-v16:16-v24:32-v32:32-v48:64-v96:128-v192:256-v256:256-v512:512-v1024:1024"
+target triple = "spir64"
+
+; Function Attrs: convergent noinline nounwind optnone
+define spir_kernel void @foo(float %a, float addrspace(1)* %b) #0 !kernel_arg_addr_space !4 !kernel_arg_access_qual !5 !kernel_arg_type !6 !kernel_arg_base_type !6 !kernel_arg_type_qual !7 {
+entry:
+  %a.addr = alloca float, align 4
+  %b.addr = alloca float addrspace(1)*, align 8
+  store float %a, float* %a.addr, align 4
+  store float addrspace(1)* %b, float addrspace(1)** %b.addr, align 8
+  %0 = load float, float* %a.addr, align 4
+  %neg = fneg float %0
+  %1 = load float addrspace(1)*, float addrspace(1)** %b.addr, align 8
+  store float %neg, float addrspace(1)* %1, align 4
+  ret void
+}
+
+attributes #0 = { convergent noinline nounwind optnone "correctly-rounded-divide-sqrt-fp-math"="false" "denorms-are-zero"="false" "disable-tail-calls"="false" "frame-pointer"="none" "less-precise-fpmad"="false" "min-legal-vector-width"="0" "no-infs-fp-math"="false" "no-jump-tables"="false" "no-nans-fp-math"="false" "no-signed-zeros-fp-math"="false" "no-trapping-math"="false" "stack-protector-buffer-size"="8" "uniform-work-group-size"="true" "unsafe-fp-math"="false" "use-soft-float"="false" }
+
+!llvm.module.flags = !{!0}
+!opencl.ocl.version = !{!1}
+!opencl.spir.version = !{!2}
+!llvm.ident = !{!3}
+
+!0 = !{i32 1, !"wchar_size", i32 4}
+!1 = !{i32 1, i32 0}
+!2 = !{i32 1, i32 2}
+!3 = !{!"clang version 10.0.0-svn369720-1~exp1+0~20190823011559.2434~1.gbpf9fff7 (trunk)"}
+!4 = !{i32 0, i32 1}
+!5 = !{!"none", !"none"}
+!6 = !{!"float", !"float*"}
+!7 = !{!"", !""}
+
+; RUN: llvm-as %s -o %t.bc
+; RUN: llvm-spirv %t.bc -o %t.spv
+; RUN: spirv-val %t.spv
+; RUN: llvm-spirv %t.spv --to-text -o - | FileCheck %s --check-prefix=CHECK-SPIRV
+
+; CHECK-SPIRV: FNegate
```

Comment:
Could you please move this to the beginning of the file?

Reply:
Done